tools/bt2img.py

```diff
@@ -91,15 +100,19 @@ def parse_command_line():
                         help='Generate a legend image as well')
     return parser.parse_args()
 
-# An XML file can have multiple behavior trees defined in it in theory. We don't
-# currently support that.
-def find_behavior_tree(xml_tree):
+def find_root_tree_name(xml_tree):
+    return xml_tree.getroot().get('main_tree_to_execute')
```

Comment:
Is this a standard name other things use? This seems like it should be a configuration to select. Why can't the trees find the main tree? If you have them including each other, you should be able to determine the root tree without having to hard code it.

Reply:
Yep, I understand your point. I did it like that, because the bt library itself uses also the `main_tree_to_execute` attribute to get the main tree. The reason for that I don‘t know.
Maybe if one wants to have multiple trees independent of each other in the same file.
If you want it different, I can look at it, but if bt lib already depends on it...

Reply:
That would be fine, but none of our existing BTs use it (or do they?). I think you should spend a little time and figure out if you can find the root tree without having to check for some hardcoded value. I think you should be able to.

Reply:
Yes, they do 😉
As I said, bt lib depends on it.
But I will look at it.

Reply:
Oh, in that case, this is OK.